docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+app:
+  build: ./
+  volumes:
+    - ./:/code
+  links:
+    - database
+  working_dir: /code
+
+database:
+  image: mysql:5
```

Comment:
Shouldn't we pin this to the version we're using on ours servers, i.e. 5.6? The chance of inadvertently using a new feature is slim, but we can guard against it.